src/javascripts/views/startApp.js

```diff
@@ -0,0 +1,23 @@
+import domBuilder from '../components/forms/domBuilder';
+import navBar from '../components/forms/navBar';
+import getFood from '../helpers/foodData';
+import { showFood } from '../components/food';
+// import loginButton from '../components/loginButton';
```

Comment:
Do you plan on using this commented-out code later?

Reply:
Yeah, we will be using this code later. We were just commenting it out to make things work while merging locally

Reply:
😎 Just making sure!

Reply:
Np! Thanks for checking!